content/docs/reference/reference.json

```diff
@@ -135,22 +135,101 @@
     "type": "string",
     "short_description": "Base64-encoded string or relative file location"
   },
-  "certificate-authority": {
-    "id": "certificate-authority",
-    "title": "Certificate Authority",
-    "path": "/certificates",
-    "description": "Certificate Authority is set when behind-the-ingress service communication uses self-signed certificates.",
-    "services": [],
-    "type": "string",
-    "short_description": "Base64 encoded string or relative file location"
-  },
   "identity-provider-settings": {
     "id": "identity-provider-settings",
     "title": "Identity Provider Settings",
     "path": "/identity-provider-settings",
     "description": "Configure and self-host your own Identity Provider with Pomerium's Identity Provider settings.",
     "type": "string"
   },
+  "cookie-secure": {
+    "id": "cookie-secure",
+    "title": "Secure Cookie",
+    "path": "/cookie-secure",
+    "description": " If true, instructs browsers to only send user session cookies over HTTPS.",
+    "services": [],
+    "type": "bool"
+  },
+  "https-only": {
+    "id": "https-only",
+    "title": "HTTPS only",
+    "path": "/cookies#cookie-http-only",
+    "description": "If true, instructs browsers to only send user session cookies over HTTPS.",
+    "services": [],
+    "type": "bool"
+  },
+  "identity-provider": {
+    "id": "identity-provider",
+    "title": "Identity Provider",
+    "path": "/../enterprise/identity-providers",
+    "description": "Choose from a list of supported IDPs.",
+    "type": "string",
+    "enterpriseOnly": true
+  },
+  "idp-options-auth0": {
+    "id": "idp-options-auth0",
+    "path": "/../enterprise/identity-providers/auth0",
```

Comment:
```suggestion
    "path": "/docs/identity-providers/auth0",
```

Reply:
actually you don't need the /docs part with these